cafebabel/templates/articles/drafts/create.html

```diff
@@ -1,14 +1,20 @@
 {% extends "base.html" %}
 {% import "articles/macros.html" as macros without context %}
+{% from "_macro-extra-js.html" import extra_js without context %}
 
-{% block extra_js %}<script src={{ url_for('static', filename='js/articles-tags.js') }}></script>{% endblock %}
+{% block extra_js %}
+  {{ extra_js(url_for('static', filename='js/noise.js')) }}
+  {{ extra_js(url_for('static', filename='js/articles-tags.js')) }}
+  {{ extra_js(url_for('static', filename='js/image-preview.js')) }}
+{% endblock %}
 
 {% block body_class %}create-article-page{% endblock %}
 {% block body %}
   <form method=post action={{ url_for('drafts.create') }} enctype=multipart/form-data>
-  <canvas class=noise></canvas>
   <div class=file>
-    <label for=image>Click here to add a cover picture or Drag & drop your pic here <small>Recommended file size: less than 1Mo</small></label>
+    <canvas class=noise></canvas>
+    <output></output>
+    <label for=image>Click here to add a cover picture or Drag & drop your pic here <small>Recommended file size: 1200x467px, less than 1Mo</small></label>
```

Comment:
Wondering how we should round this for the end user. 1200x500px?

Reply:
Previous website picture format was 1180x461px. That's why I would like to keep the proportions. What do you think about that @vinyll ?

Reply:
For sure you're right that we should keep the proportions for the existing articles.
My question is more about the dimensions we ask the user to make for new articles; asking the user to make a picture 467px height is so precise that it sounds restrictive. Using a rounded value like 500px (or eventually 450px) could ease that feeling.
But it may be interesting to just quickly ask the editors around instead ;)

Reply:
Indeed, it would be easier for editors. But if we write a velue like 500px, then we will get images with a 500px height. Won't fit previous format then ? I don't get it.